demisto_sdk/commands/openapi_codegen/openapi_codegen.py

```diff
@@ -670,6 +670,9 @@ def extract_outputs(self, data: dict) -> tuple:
                     for ref in refs:
                         ref = ref.split('/')[-1]
                         ref_props = self.extract_values(self.reference.get(ref, {}), 'properties')
+                        # Addition of filtering dicts only was added because some swaggers contain example files
+                        # Which are written in string and caused errors on ref_props[0].items()
+                        ref_props = [ref_prop for ref_prop in ref_props if isinstance(ref_prop, dict)]

```

Comment:
is this the only area in the code where this could happen? please have a look at line 720 and other similar areas in the code, and make sure we cover all these cases

Reply:
I added to line 720 too as it seems relevant. The code is very complex to be confident I handled all those cases, but I did double check now and have not found any other place which seems relevant for this specific fix